backend/src/test/java/ca/gov/dtsstn/cdcp/api/web/json/JsonPatchHttpMessageConverterTests.java

```diff
@@ -66,4 +82,17 @@ void testWriteInternal() throws Exception {
 		assertThat(byteArrayOutputStream).hasToString("[{\"op\":\"replace\",\"path\":\"/id\",\"value\":\"value\"}]");
 	}
 
+	@Test
+	@DisplayName("Test writeInternal(..) with flawed input")
+	void testWriteInternalWithFlawedInput() throws Exception {
+		final var httpOutputMessage = mock(HttpOutputMessage.class);
+		final var byteArrayOutputStream = new ByteArrayOutputStream();
+
+		when(httpOutputMessage.getBody()).thenThrow(new IOException("Something went wrong"));
+
+		final var map = Map.of("op", "replace", "path", "/id", "value", "value");
+		final var jsonPatchObject = Json.createArrayBuilder(List.of(map)).build();
+		assertThrows(HttpMessageNotWritableException.class, () -> jsonPatchHttpMessageConverter.writeInternal(Json.createPatch(jsonPatchObject), httpOutputMessage));
```

Comment:
I think there is a better way to create a patch object.. take a look at `JsonPatchProcessorTests` around line 71,

Reply:
Okay, `createPatchBuilder` is now used.